### DISCUSSION TITLE
BoundingBoxes related to canvas

### DISCUSSION THREAD

**Discussion by SonahtQ** [fabricjs/fabric.js#7823] [Q&A] [answered]
How can I get global bounding box dimensions (left, top, widht, height) of object? Global means that dimensions must be related to canvas. I have a lot of objects and groups with objects - rotated, scaled and skrewed. Now each object and each group and also each object in group has to have borders rendered on canvas, borders of bounding box/rect. Just for visualization.

**Comment by melchiar** [comment]
http://fabricjs.com/bounding-rectangle

**Comment by SonahtQ** [comment]
Yes, i've saw that example. But it doesn't work for objects in group, returned values aren't related to canvas.

**Reply by melchiar** [reply]
you need to calculate the transform matrix to get values for objects inside a group. see https://github.com/fabricjs/fabric.js/issues/7407

**Comment by ShaMan123** [comment, marked as answer]
This is about to be solved by #7859. Take a look https://github.com/fabricjs/fabric.js/pull/7859/files#diff-40b491107aa7c801299280821dc9af3ba6b15ebe42f72a3942054bdaf8086d8d
